net.nokyan.Resources.json

```diff
@@ -0,0 +1,139 @@
+{
+    "id": "net.nokyan.Resources",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-nightly",
+        "org.freedesktop.Sdk.Extension.llvm16"
+    ],
+    "command": "resources",
+    "finish-args": [
+        "--device=dri",
+        "--env=G_MESSAGES_DEBUG=none",
+        "--env=RUST_BACKTRACE=full",
+        "--env=RUST_LOG=resources=debug",
+        "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/usr/share:/run/host/share:/var/lib/flatpak/exports/share:~/.local/share/flatpak/exports/share",
+        "--filesystem=/var/lib/flatpak/app:ro",
+        "--filesystem=/var/lib/flatpak/exports/share:ro",
+        "--filesystem=xdg-data/flatpak/app:ro",
+        "--filesystem=xdg-data/flatpak/exports/share:ro",
+        "--filesystem=host",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-nightly/bin:/usr/lib/sdk/llvm16/bin",
+        "env": {
+            "CARGO_REGISTRIES_CRATES_IO_PROTOCOL": "sparse",
+            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER": "clang",
+            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/app/bin/mold",
+            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER": "clang",
+            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/app/bin/mold"
+        },
+        "test-args": [
+            "--socket=x11"
+        ]
+    },
+    "modules": [
+        {
+            "name": "glib",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dtests=false",
+                "-Dselinux=disabled"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/glib/2.78/glib-2.78.0.tar.xz",
+                    "sha256": "44eaab8b720877ce303c5540b657b126f12dc94972d9880b52959f43fb537b30"
+                },
+                {
+                    "type": "patch",
+                    "path": "0001-do-not-test-exec.patch"
+                }
+            ]
+        },
+        {
+            "name": "mold",
```

Comment:
why do you need to build mold?

Reply:
It's not included with rust nightly for some reason.

Reply:
I'll look into it.